### DISCUSSION TITLE
Console Not Running

### DISCUSSION THREAD

**Discussion by mulyadi1994** [lxdware/lxd-dashboard#40] [Q&A]
Hi I I'm using Version: v3.4.0, i check  in web console is not running, 

![image](https://user-images.githubusercontent.com/8446243/170188183-34421f21-51a9-4ddb-bc9f-ebb1931625c5.png)

this is my php log : 
[24-May-2022 05:56:13] NOTICE: exiting, bye-bye!
[24-May-2022 05:58:46] NOTICE: fpm is running, pid 121
[24-May-2022 05:58:46] NOTICE: ready to handle connections
[24-May-2022 05:58:46] NOTICE: systemd monitor interval set to 10000ms

and i dont have any error log in my nginx log's.

note: i have trusted the self sign cert to my LXD

**Comment by matthewalanpenning** [comment]
Interesting. Does Exec work? What is the container operating system? I have not yet been able to replicate the issue but it appears as though the browser is having an issue getting a ready state from the websocket connection.

**Comment by ingobaab** [comment]
I replaced the LXD's server.key and server.crt with letsencrypt certificates, then both, Console and Exec did work (in Chrome).
